File: codewars/complementary-dna/complementary-dna.cpp
#include <string>

std::string DNAStrand(const std::string& dna)
{
  //your code here
  std::string comp = "";
  for (unsigned long i = 0; i < dna.size(); i++) {
    if (dna[i] == 'A') {
      comp += 'T';
    } else if (dna[i] == 'T') {
      comp += 'A';
    } else if (dna[i] == 'C') {
      comp += 'G';
    } else {
      comp += 'C';
    }
  }
  return comp;
}

File: codewars/complementary-dna/complementary-dna.py
def DNA_strand(dna):
    # code here
    comp = ''
    
    for char in dna:
        if char == 'A': comp += 'T'
        elif char == 'T': comp += 'A'
        elif char == 'G': comp += 'C'
        else: comp += 'G'
        
    return comp